### ISSUE TITLE
avoid overflow in dialog box on big sur

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#8524]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.dev
    OS Version      : macOS 11.0.1 (Big Sur)
    R Version       : Any

### Steps to reproduce the problem

Load and then attempt to update an R package. For example:

```
library(MASS)
install.packages("MASS")
```

### Describe the problem in detail

Shared from Slack:

<img width="280" alt="Screenshot 2020-12-03 at 10 37 04" src="https://user-images.githubusercontent.com/1976582/101068440-97c5ab00-354d-11eb-81b7-0c851d5dc683.png">

### Describe the behavior you expected

No overflowing on dialog boxes.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Verified on Big Sur 11.0.1 with RStudio 1.4.1077:

![2020-12-03_15-30-55](https://user-images.githubusercontent.com/10569626/101101093-a7101d00-357c-11eb-9929-8dccbb050246.png)
